src/v3/src/components/List/List.tsx

```diff
@@ -14,7 +14,8 @@ import { List as ListOdyssey } from '@okta/odyssey-react';
 import { Box, Typography } from '@okta/odyssey-react-mui';
 import { h } from 'preact';
 
-import { ListElement, UISchemaElementComponent } from '../../types';
+import { ListElement, UISchemaElementComponent, UISchemaLayout } from '../../types';
+import Layout from '../Form/Layout';
```

Comment:
This import is causing a dependency cycle 😭

Reply:
You should be able to use https://github.com/okta/okta-signin-widget/blob/0.1/src/v3/src/components/Form/LayoutContainer.tsx to get around the dependency cycle I think?

Reply:
This fixed some of the issues, but not all. The issue is that List contains ElementContainer which contains renderers... which contains List :/

Reply:
Oh nooo, yea I recall this when trying to resolve the original dependency cycle issue :(